config/webpack/webpack.config.dev.js

```diff
@@ -1,24 +1,20 @@
 "use strict";
 
-var webpack = require("webpack");
 var config = require("./webpack.config");
 var LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
 
 // **WARNING**: Mutates base configuration.
 // We do this because lodash isn't available in `production` mode.
 config.output.filename = config.output.filename.replace(/\.min\.js$/, ".js");
-config.output.pathinfo = true;
```

Comment:
Interesting. `mode: development` enables this anyways. The real way to disable is:

```js
config.output.pathinfo = false;
```

I'm OK with adding `false` or keeping this change as-is ("do the default thing").